src/tests/SearchView.test.jsx

```diff
@@ -438,6 +497,15 @@ describe('Search View', () => {
 
     const searchInput = getByRole('textbox');
     await user.type(searchInput, 'terryjones@python.com');
+    jest.runOnlyPendingTimers();
+
+    expect(spyOnFetch).toHaveBeenCalledWith(
+      'https://treeherder.mozilla.org/api/project/autoland/push/?author=terryjones@python.com',
+    );
+
+    await waitFor(() => expect(queryByText('try')).not.toBeInTheDocument());
```

Comment:
You could use `waitForElementToBeRemoved` as well, but it's up to you! [more information](https://testing-library.com/docs/guide-disappearance#waiting-for-disappearance)